teaching.rkt

```diff
@@ -193,6 +198,7 @@
                  }
                  @li{
                      @strong{4400} Programming Languages @br{}
+                     @indent[] S 2020 @a[href: "https://felleisen.org/matthias/4400-s20/" target: "_blank"]{link} @instructor{Matthias Felleisen} @br{}
```

Comment:
Apparently the course is called Principles of Programming Languages.

Reply:
Hm, on Registrar it's called Programming Languages
![Screenshot_20200502_103134_org mozilla firefox](https://user-images.githubusercontent.com/8005679/80867177-b2a30a80-8c60-11ea-967e-77985f8201c0.jpg)

Reply:
OK. (Everyone refers to this thing as PPL and the PhD level as iPPL.)

Reply:
Maybe somebody could make a name change request! :)